package.json

```diff
@@ -75,28 +75,32 @@
     "nyc": "^14.0.0",
     "prettier": "^1.17.0",
     "react-dev-utils": "^9.0.0",
-    "react-testing-library": "^7.0.0",
     "redux-mock-store": "^1.5.3",
     "style-loader": "^0.23.1",
     "webpack": "^4.30.0",
     "webpack-cli": "^3.3.1",
     "webpack-dev-server": "^3.3.1"
   },
   "dependencies": {
+    "@yaireo/tagify": "^2.18.3",
     "axios": "^0.18.0",
     "css-loader": "^2.1.1",
     "express": "^4.16.4",
     "jquery": "^3.4.1",
     "jwt-decode": "^2.2.0",
+    "medium-editor": "^5.23.3",
+    "medium-editor-insert-plugin": "^2.5.1",
     "moment": "^2.24.0",
     "node-sass": "^4.12.0",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-content-loader": "^4.2.1",
     "react-dom": "^16.8.6",
+    "react-medium-editor": "^1.8.1",
     "react-redux": "^7.0.3",
     "react-router-dom": "^5.0.0",
     "react-tabs": "^3.0.0",
+    "react-testing-library": "^7.0.0",
```

Comment:
I thought `react-testing-library` is a dev-dependency?

Reply:
@chikeozulumba you may consider adding "import/no-extraneous-dependencies": 0 to eslintrc file to prevent eslint from complaining about react-testing-library being in dev-dependency rather than dependency.